scripts/trn_condensed.R

```diff
@@ -0,0 +1,129 @@
+# Script to condense duplicate entries in trn_trn table to make manual reconciliation easier.
+
+library(here)
+library(fs)
+library(lubridate)
+library(tidyverse)
+library(ctregistries)
+
+# Load long trn_trn table
+trn_trn <- read_rds("data/cross-registrations/trn_trn.rds")
+
+# Drop unnecessary columns, add new publication booleans
+trn_trn_no_regs <- trn_trn |>
+  select(-registry1, -registry2)
+
+# Create a unique identifier for pairs, which allows us to identify duplicate pairs
+trn_trn_pair_id <- trn_trn_no_regs %>%
```

Comment:
Please use the R native pipe |> for consistency, and also rowwise has to be followed by ().
```suggestion
trn_trn_pair_id <- trn_trn_no_regs |>
rowwise() |>
```
etc...